toqito/matrix_props/is_totally_positive.py

```diff
@@ -19,7 +20,7 @@ def is_totally_positive(mat, tol=1e-6, sub_sizes=None):
         \begin{pmatrix}
             1 & 2 \\
             3 & 4
-        \end{pmatrix}
+        \\end{pmatrix}
```

Comment:
Hmm, is there are reason there are two `\\` here?

Reply:
For some reason, `isort` or `black` made this change. I'll check it out later.